hokusai/lib/docker_compose_helpers.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+import yaml
+
+from hokusai import CWD
+from hokusai.lib.config import HOKUSAI_CONFIG_DIR
+from hokusai.lib.exceptions import HokusaiError
+from hokusai.lib.template_selector import TemplateSelector
+from hokusai.services.yaml_spec import YamlSpec
+
+def follow_extends(docker_compose_yml):
+  with open(docker_compose_yml, 'r') as f:
+    rendered_templates = []
+    try:
+      struct = yaml.safe_load(f.read())
+      for service_name, service_spec in struct['services'].iteritems():
+        extended_filename = service_spec['extends']['file']
```

Comment:
Perhaps this isn't great implicit control flow - this line will raise a `KeyError` if the spec does not contain an extends key and the `except` block below will swallow it

Reply:
Fixed in 32177c5ccc46b7c10cac18b443d97bdc3e82ce70